apps/vocabulary/main.go

```diff
@@ -0,0 +1,261 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// report is a demo application that displays information about an
+// OpenAPI description.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
+
+	pb "github.com/googleapis/gnostic/openapiv2"
+)
+
+func readDocumentFromFileWithName(filename string) (*pb.Document, error) {
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+	document := &pb.Document{}
+	err = proto.Unmarshal(data, document)
+	if err != nil {
+		return nil, err
+	}
+	return document, nil
+}
+func addToMap(word string, mapName map[string]int) {
+	_, ok := mapName[word]
+	if ok {
+		mapName[word] += 1
+	} else {
+		mapName[word] = 1
+	}
+}
+
+func createCSV(names map[string]int) {
+	f, ferror := os.Create("IndexFreqSort.csv")
```

Comment:
Since we've stopped calling this, let's take it out before committing (it might be worth hanging onto this code for use in a future helper app)